lib/command_graph.c

```diff
@@ -0,0 +1,480 @@
+/*
```

Comment:
https://github.com/FRRouting/frr/blob/master/COMMUNITY.md#file-header-required-for-new-files-added